invenio/base/scripts/database.py

```diff
@@ -77,17 +77,25 @@ def init(user='root', password='', yes_i_know=False):
         cmd_admin_prefix = prefix.format(cmd='mysqladmin', user=user,
                                          password=password,
                                          **args)
+
+        # we can't wrap all keys in quotes since the table name for instance is not quotable'
+        keys_to_wrap =['CFG_DATABASE_PASS', 'CFG_DATABASE_USER']
```

Comment:
It's fine as it is, however @hachreak is working on some improvements https://github.com/hachreak/invenio/commits/refactor_database_init.